README.md

```diff
@@ -222,7 +222,8 @@ The following table lists the configurable parameters of the Harbor chart and th
 | `core.secretName`                                                           | Fill the name of a kubernetes secret if you want to use your own TLS certificate and private key for token encryption/decryption. The secret must contain keys named: `tls.crt` - the certificate and `tls.key` - the private key. The default key pair will be used if it isn't set                                                                                                                                                                                                           |                                                                                     |
 | `core.xsrfKey`                                                              | The XSRF key. Will be generated automatically if it isn't specified                                                                                                                                                                                                                                                                                                                                                                                                                            |                                                                                     |
-| `core.priorityClassName`                                                    | The priority class to run the pod as                                                                                                                                                                                                                                                                                                                                                                                                                                                           |                                                                                     |
+| `core.priorityClassName`                                             | The priority class to run the pod as                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
```

Comment:
each row should contains 4 "|", but `core.priorityClassName` only contains 3 "|" in this PR

Reply:
> each row should contains 4 "|", but `core.priorityClassName` only contains 3 "|" in this PR

updated